src/filefinder.cpp

```diff
@@ -26,6 +26,7 @@
 #include <string>
 #include <vector>
 #include <sstream>
+#include <regex>
```

Comment:
unnecessary include (thanks for changing this :+1: )

Reply:
Fixed (and you're welcome!)

Reply:
Also fixed the commit message.